plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridTableBase.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.model;
+
+import java.util.Collection;
+import java.util.Map;
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.Log;
+import org.jkiss.dbeaver.ext.cubrid.CubridConstants;
+import org.jkiss.dbeaver.ext.generic.model.GenericTableBase;
+import org.jkiss.dbeaver.model.DBPEvaluationContext;
+import org.jkiss.dbeaver.model.DBUtils;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils;
+import org.jkiss.dbeaver.model.meta.IPropertyValueListProvider;
+import org.jkiss.dbeaver.model.meta.Property;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+import org.jkiss.dbeaver.model.struct.DBSObject;
+
+public abstract class CubridTableBase extends GenericTableBase{
+	
+    private static final Log log = Log.getLog(CubridTableBase.class);
+
+	private CubridUser owner;
+	private CubridObjectContainer container;
+	private boolean isSystemTable;
+
+	public CubridTableBase(CubridObjectContainer container, String tableName, String tableType, JDBCResultSet dbResult) {
+		super(container, tableName, tableType, dbResult);
+		this.container = container;
+		String owner_name;
+
+		if (dbResult != null) {
+			owner_name = JDBCUtils.safeGetString( dbResult, CubridConstants.OWNER_NAME);
+			isSystemTable = JDBCUtils.safeGetString(dbResult, CubridConstants.IS_SYSTEM_CLASS).equals("YES");
+		} else {
+			owner_name = getDataSource().getContainer().getConnectionConfiguration().getUserName().toUpperCase();
+			isSystemTable = false;
+		}
+
+		for(CubridUser cbOwner : getUsers()){
+			if(cbOwner.getName().equals(owner_name)) {
+				this.owner = cbOwner;
+				break;
+			}
+		}
+	}
+
+	public boolean isPhysicalTable() {
+		return !isView();
+	}
+
+	@Override
+	public boolean isSystem() {
+		return isSystemTable;
+	}
+	public CubridObjectContainer getContainer() {
+		return this.container;
+	}
+
+	public Collection<? extends CubridUser> getUsers() {
+		try {
+			return container.getDataSource().getCubridUsers(null);
+		} catch (DBException e) {
+			log.error("Cannot get user.");

```

Comment:
It's better to add an exception to `log.error` when caught

Reply:
I have fixed it.